compiler/src/dotty/tools/dotc/config/Settings.scala

```diff
@@ -88,7 +88,7 @@ object Settings {
         state.update(idx, x.asInstanceOf[T])
     }
 
-    def isDefaultIn(state: SettingsState) = valueIn(state) == default
+    def isDefaultIn(state: SettingsState): Boolean = valueIn(state) == default
 
```

Comment:
I guess we don’t need to add this type annotation, no? Otherwise, LGTM.

Reply:
I followed the Boy Scout Rule and added this small thing :) Is it true that the public methods have to have the return type annotation? Please correct me if I am not right.